libraries/AP_Volz_Protocol/AP_Volz_Protocol.cpp

```diff
@@ -127,37 +201,9 @@ void AP_Volz_Protocol::send_command(uint8_t data[VOLZ_DATA_FRAME_SIZE])
     }
 
     // add CRC result to the message
-    data[4] = HIGHBYTE(crc);
-    data[5] = LOWBYTE(crc);
-    port->write(data, VOLZ_DATA_FRAME_SIZE);
-}
-
-void AP_Volz_Protocol::update_volz_bitmask(uint32_t new_bitmask)
-{
-    uint8_t count = 0;
-    last_used_bitmask = new_bitmask;
-
-    for (uint8_t i=0; i<NUM_SERVO_CHANNELS; i++) {
-        if (new_bitmask & (1U<<i)) {
-            count++;
-        }
-    }
-
-    // have a safety margin of 20% to allow for not having full uart
-    // utilisation. We really don't want to start filling the uart
-    // buffer or we'll end up with servo lag
-    const float safety = 1.3;
-
-    // each channel take about 425.347us to transmit so total time will be ~ number of channels * 450us
-    // rounded to 450 to make sure we don't go over the baud rate.
```

Comment:
I'm not sure where 425 is from, the correct time is 520, see:
https://github.com/IamPete1/ardupilot/blob/d4f30458b8b19865b5a9012a71b123ec16257e08/libraries/AP_Volz_Protocol/AP_Volz_Protocol.cpp#L89-L91

Double checked with the Saleae:
![image](https://github.com/ArduPilot/ardupilot/assets/33176108/828ed468-18da-4a2c-9ee4-0addf7da3dc9)

The round up to 450 and the 1.3 safety factor puts the actual number used as 585.

Reply:
That's correct. Typical baudrate for Volz protocol is 115200bps, it makes 10bit*6bytes/115200 = 521us